task3/script.js

```diff
@@ -11,5 +11,5 @@ for (let i =1; i <=2; i++) {
     ul.appendChild(li).appendChild(a).insertAdjacentElement('afterbegin', img);
 }
 
-const firstChild = ul.firstChild;
-
+ul.firstChild.querySelector('img').src = "/img/bookmark.png";
+ul.lastChild.querySelector('img').src = "/img/message.png";
```

Comment:
![image](https://user-images.githubusercontent.com/6824305/130657612-30ccabf0-4175-4a6c-9291-d16a146b2713.png)

There's a small detail.
There is a way to get rid of the last line that appears like this

I would like to quote from a previous review by another teacher.


[
https://github.com/csakyo/homework/pull/15#pullrequestreview-676777039](https://github.com/csakyo/homework/pull/15#pullrequestreview-676777039)

Reply:
I didn't notice the line coming in at the end.

I changed the settings in a text editor.
Thank you for pointing out the small details.

Reply:
This one is also OK.